app/static/js/volunteerProfile.js

```diff
@@ -113,6 +113,18 @@ $(document).ready(function(){
       }
     })
   });
+  // Popover functionalitie
+  $(function(){
+    var requiredTraining = $(".trainingPopover");
+    requiredTraining.popover({
+      trigger: "hover",
+      sanitize: false,
+      html: true,
+      content: function() {
+                     return $(this).attr('data-content');
+        }
+    });
+  });
```

Comment:
When is this called? I'm confused why it is in the jquery function

Reply:
We need this to actually display the trainings if I'm not mistaken, the return happens upon hover.

Reply:
What I mean is, is this supposed to be in the onready handler, instead of in `$(function() { })`. Clearly it works, I'm just a little confused as to how. I don't know when this function runs